### ISSUE TITLE
Invallid box drawing in Windows

### ISSUE THREAD

**Issue by dmitry-zhuravlev** [cbeust/kobalt#218]
Some cosmetic issue when running Kobalt tasks on Windows:

Running task with IDEA plugin cause right edge of the box becomes shifted to the left:
![image](https://cloud.githubusercontent.com/assets/6297697/15706971/04a79d5c-27ff-11e6-87b2-4649cae9b374.png)

But running the same task within Windows terminal looks different:
![image](https://cloud.githubusercontent.com/assets/6297697/15707031/3a52ef10-27ff-11e6-9d7f-46054ba2d448.png)

@cbeust Any thoughts about that?

**Comment by cbeust** [comment]
I noticed that too, it's been around forever, never bothered trying to
figure it out.

The only place where I've noticed it's wrong is in IDEA's shell window, so
I just thought it's a bug in their console management.

-- 
Cédric

**Comment by dmitry-zhuravlev** [comment]
The second screenshot was from terminal embedded in IDEA. Maybe it is a different fonts problem....

**Comment by cbeust** [comment]
Yes. Not sure it's worth spending time on this as long as it's limited to
IDEA's console window.

-- 
Cédric
